gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/publisher/GobblinMCEPublisher.java

```diff
@@ -214,6 +214,7 @@ public static Metrics getMetrics(State state, Path path, Configuration conf, Nam
           return OrcMetrics.fromInputFile(HadoopInputFile.fromPath(path, conf), MetricsConfig.getDefault(), mapping);
         } catch (Exception e) {
           //This means the table is not compatible with iceberg, so return a dummy metric
+          //todo: throw exception here? as it's corrupted file
```

Comment:
These two comments seem to be contradictory it kind of sounds like. If the table is not compatible with iceberg, then rather than return a dummy metric (which indicates the table is not compatible but no underlying issues), we should just throw the exception due to corrupted file right?

Reply:
I remove the catch exception part as I verified that the only exception is because of we cannot open the file, which either indicating we have issue on HDFS or means it's a corrupted file. We should be safe to just throw exception and fail the task in this case.